subprojects/docs/src/docs/release/notes.md

```diff
@@ -52,6 +52,11 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Tooling API deprecation
```

Comment:
This title sounds scary :) Consider something like "Running very old Gradle versions via the Tooling API deprecated"